app/src/main/java/comcmput301f16t01/github/carrier/MainActivity.java

```diff
@@ -274,18 +286,26 @@ private void fillRiderRequests(ListView requestListView) {
             requestListView.setAdapter( requestArrayAdapter );
 
             final Context ctx = this.getContext();
-            requestListView.setOnItemLongClickListener(new AdapterView.OnItemLongClickListener() {
+            /*requestListView.setOnItemLongClickListener(new AdapterView.OnItemLongClickListener() {
```

Comment:
This was something Bennett wanted to do originally but I think we agreed to not do it. I didn't want to delete the code fully.